src/validators.ts

```diff
@@ -224,3 +224,25 @@ export function max(maximum: number, message?: string): Validator {
     execute: executeMax(maximum)
   }
 }
+
+const WORD_REGEX = /^[a-zA-Z]+$/
+
+export function word(message?: string): Validator {
```

Comment:
I'd suggest naming this `alpha` instead, it would seem more appropriate since this is a test for only letters of the alphabet.